scalastyle-config.xml

```diff
@@ -1,127 +0,0 @@
-<scalastyle commentFilter="enabled">
```

Comment:
I was annoyed with the formatted violating the long lines by 1 character and it doesn't exist on the branch.